apps/web/pages/event-types/index.tsx

```diff
@@ -603,11 +603,20 @@ const CreateFirstEventTypeView = () => {
 };
 
 const CTA = () => {
+  const { t } = useLocale();
+
   const query = trpc.viewer.eventTypes.getByViewer.useQuery();
 
   if (!query.data) return null;
 
-  return <CreateEventTypeButton canAddEvents={true} options={query.data.profiles} />;
+  return (
+    <CreateButton
+      subtitle={t("create_event_on").toUpperCase()}
```

Comment:
Is this intentional to change this text from 'Create an event type under your name or a team.' to 'CREATE EVENT ON'.

Reply:
Yes, it is. I just tried to find the Figma design where I saw that but I can't find it anymore. @Jaibles what is the current most recent design for that? This is how it looks with my changes: 
![Screenshot 2023-02-20 at 13 35 09](https://user-images.githubusercontent.com/30310907/220179165-46726da2-e592-4135-ab99-7e49e1458c00.png)

Reply:
We should use CSS to uppercase then.